Core.UnitTests/MutableReflection/MutableTypeTest.cs

```diff
@@ -260,12 +269,12 @@ public void AddNestedType_WithNullBaseType ()
       var typeName = "NestedType";
       var typeAttributes = TypeAttributes.NestedFamily;
       var nestedTypeFake = MutableTypeObjectMother.Create ();
-      _mutableMemberFactoryMock.Expect (mock => mock.CreateNestedType (_mutableType, typeName, typeAttributes, baseType: null)).Return (nestedTypeFake);
```

Comment:
Since Moq's `Setup` accepts an expression and not a delegate and "an expression tree may not contain a named argument specification" that unfortunately isn't possible.

https://stackoverflow.com/questions/10131772/why-cant-an-expression-tree-contain-a-named-argument-specification